jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MultiPart.java

```diff
@@ -1186,6 +1186,7 @@ private boolean parsePreamble(ByteBuffer buffer)
 
         private State parseHeaderStart(ByteBuffer buffer)
         {
+            boolean crFlag = false;
```

Comment:
I don't think this can be a local variable.   The CR might be the last byte of one chunk and the LF the first byte of the next chunk.

Reply:
Done